### DISCUSSION TITLE
New UI Broke Autograding Workflow Test Result Preview

### DISCUSSION THREAD

**Discussion by chrissphinx** [community/community#51872] [GitHub Education]
<img width="1251" alt="image" src="https://user-images.githubusercontent.com/2583639/229582710-3658e2ab-22a0-4c8d-863a-1e5775967585.png">

The latest UI change to GitHub Classroom has seemingly broken this functionality of the autograder test result being reflected by `submission-icon-container`. It always displays a check mark now, despite this specific workflow failing, clicking the element brings me to this page:

<img width="858" alt="image" src="https://user-images.githubusercontent.com/2583639/229584200-a9b6e186-397c-4dde-8301-1de2503e566f.png">

**Comment by samus-aran** [comment]
Hi @chrissphinx 👋 

Thank you for bringing this to our attention. We've passed this along to our internal team to investigate and when we have any updates we'll update this thread. 👍

**Comment by chrissphinx** [comment]
Thank you @samus-aran, does the GitHub Classroom team have a public issue tracker that can be used instead of submitting issues about the platform here in a discussion thread?

**Comment by chrissphinx** [comment]
So I guess that's a no?

**Comment by chrissphinx** [comment]
This is actually since resolved, so I will close it. :) Thank you!
